dashboard/app/views/levels/_summary.html.haml

```diff
@@ -11,6 +11,7 @@
     {
       user_id: response.user_id,
       text: response.level_source&.data,
+      student_name: User.find_by(id: response.user_id)&.name,
```

Comment:
Try this? I think it would probably work just as well.

```suggestion
      student_name: response.user.name,
```

Reply:
Also...any thoughts about putting family names in here too? More of a product question than an engineering one.

Reply:
That's it - I tried examining the full "response" but didn't see user connected to it.  It looks like it might have been because there is a relationship between `User` and `Response` - I will research this a bit more as I learn more about the nuts / bolts / magic of ruby.

Reply:
And, re "family names" - "name" includes full name which is what the spec wants so we should be good with that!

Reply:
> And, re "family names" - "name" includes full name which is what the spec wants so we should be good with that!

I don't actually think this is true. See my Rails console screenshot in my longer comment from a moment ago. `name` is the original name field, what we now refer to as "display name" in the UI. `family_name` is separate and you'll need to query for both if you want to display both.

Reply:
Youre right - thanks for checking me here!  This is updated